docs/pages/enterprise/sso/ssh-sso.mdx

```diff
@@ -182,14 +182,6 @@ $ tsh --proxy=proxy.example.com login --auth=okta
 $ tsh --proxy=proxy.example.com login --auth=local --user=admin
 ```
 
-Refer to the following guides to configure authentication connectors of both
-SAML and OIDC types:
-
-* [SSH Authentication with Okta](ssh-okta.mdx)
-* [SSH Authentication with OneLogin](ssh-one-login.mdx)
-* [SSH Authentication with ADFS](ssh-adfs.mdx)
-* [SSH Authentication with OAuth2 / OpenID Connect](oidc.mdx)
```

Comment:
Why delete these?

Reply:
In that document there were two lists of links pointing to the individual SSO guides. This list removed here was deeper down missing some of the links, so I figured that only the properly updated list at the top should be sufficient.

Reply:
I see now. I'm unsure if the old links were intended to provide value for SEO purposes or not.